src/core/Akka.Remote/Configuration/Remote.conf

```diff
@@ -310,14 +310,17 @@ akka {
       trttl = "Akka.Remote.Transport.ThrottlerProvider,Akka.Remote"
     }
 
-    ### Default configuration for the Helios based transport drivers
+	# necessary to keep backwards compatibility
+	helios.tcp.transport-class = "Akka.Remote.Transport.Helios.HeliosTcpTransport, Akka.Remote.Transport.Helios"
 
-    helios.tcp {
+    ### Default configuration for the DotNetty based transport drivers
+
+    dot-netty.tcp {
```

Comment:
Yes. I've added `log-transport` value in order to add `LoggingHandler` to DotNetty pipeline. It's used to log user events on DotNetty side.